migrations/versions/856b03419a0b_created_user_model.py

```diff
@@ -0,0 +1,36 @@
+"""created User model
+
+Revision ID: 856b03419a0b
+Revises: 61c85c9431f1
+Create Date: 2021-07-02 14:33:15.114312
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '856b03419a0b'
+down_revision = '61c85c9431f1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
I would use pre-commit with black to format even migrations, so they are more readable